### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Shaddynotshady** [Imperial-lord/mapbox-flutter#15] [Q&A]
till yesterday code was working fine i made some changes according to me but then i start getting adb issue and after it android studio got updated now code is runnig but build crash as its start

its also now not working wih location 4.3.0

**Comment by Imperial-lord** [comment]
I tried running the code with latest android studio, the turn by turn seems to work fine. Could you specify the issue a bit more clearly, so that I can help you?
